### ISSUE TITLE
Get the amount of many relations

### ISSUE THREAD

**Issue by tflori** [tflori/orm#25]
Instead of writing a own query (even with query builder) it would be nice to ask the entity itself:

```php
$user->fetch('contacts')->count();
```

The same for any EntityFetcher:

```php
$em->fetch(User::class)->where('name', 'Thomas')->count();
```

**Comment by tflori** [comment]
I commited a count method for EntityFetcher here: https://github.com/tflori/orm/commit/5a8d24bcaad528b7fe32ccea472317d7caf41b78

please try out (by installing dev-feature-fetcherCount) if this solves your problem

**Comment by tflori** [comment]
solved in #27
